docs/known_issues.rst

```diff
@@ -18,3 +18,9 @@ Serial DFU stops working after using GNU screen
   **Workaround:** Use a different serial terminal application, for example, `Serial Terminal app`_, ``tio`` or, power cycle device, if needed.
 
   **Affected platforms:** Thingy:91 X
+This sample requires less hardware resources when it is run on an nRF54 Series device compared to the nRF52 or nRF53 Series devices.
```

Comment:
This section is not relevant for this project, as we only support Thingy:91 X here

Reply:
My bad, this was a copy paste error. The content was of another documentation.